contributed_definitions/nyaml/NXxps.yaml

```diff
@@ -0,0 +1,474 @@
+        depends_on(NX_CHAR):
+          doc: |
+            Link to transformations defining the XPS base coordinate system,
+            which is defined such that the positive z-axis points along the sample stage normal, and the
+            x- and y-axes lie in the plane of the sample stage.
+        (NXtransformations):
+          doc: |
+            Set of transformations, describing the orientation of the XPS coordinate system
+            with respect to the beam coordinate system (.).
+    (NXinstrument):
+      doc:
+      - |
+        Description of the XPS spectrometer and its individual parts.
+      - |
+        xref:
+          spec: ISO 18115-1:2023
+          term: 12.58
+          url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.58
+      sourceTYPE(NXsource):
+        power(NX_FLOAT):
+          unit: NX_POWER
+          exists: recommended
+        transformations(NXtransformations):
+          exists: recommended
+          source_polar_angle_of_incidence(NX_NUMBER):
+            unit: NX_ANGLE
+            doc: |
+              Incidence angle
+            \@transformation_type:
+              enumeration: [rotation]
+            \@vector:
+              enumeration: [[1, 0, 0]]
+            \@depends_on:
+              enumeration: [source_azimuth_angle]
+          source_azimuth_angle(NX_NUMBER):
+            unit: NX_ANGLE
+            doc: |
+              Azimuthal tilt of the source from the y-direction towards the operator, defined
+              by the sample stage.
+            \@transformation_type:
+              enumeration: [rotation]
+            \@vector:
+              enumeration: [[0, 0, -1]]
+            \@depends_on:
+              enumeration: [/entry/sample_stage_cs_transformations/vamas_sample_stage_coordinate_system]
+      (NXelectronanalyser):
+        work_function(NX_FLOAT):
+        transmission_function(NXdata):
+          exists: recommended
+        (NXcollectioncolumn):
+          magnification(NX_FLOAT):
+            exists: recommended
+            unit: NX_DIMENSIONLESS
+        (NXenergydispersion):
+          diameter(NX_NUMBER):
+            exists: recommended
+            unit: NX_LENGTH
+          energy_scan_mode:
+        depends_on:
+          exists: recommended
+          doc: |
+            Reference to the transformation describing the orientation of the analyzer
+            relative to a defined coordinate system.
+        transformations(NXtransformations):
+          exists: recommended
+          analyser_take_off_polar_angle(NX_NUMBER):
+            unit: NX_ANGLE
+            doc: |
+              Polar tilt of the analyser with respect to the upward z-direction, defined by
+              the sample stage.
+            \@transformation_type:
+              enumeration: [rotation]
+            \@vector:
+              enumeration: [[1, 0, 0]]
+            \@depends_on:
+              enumeration: [analyser_take_off_azimuth_angle]
```

Comment:
Is this the right order you want to have? Typically, Polar comes first, and then azimuth (i.e. azimuth is mounted ontop of polar).

Reply:
You are right, I switched the order as suggested.